From jbe@pengutronix.de  Thu Jul 19 21:57:08 2007
From: Juergen Beisert <jbe@pengutronix.de>
Date: Thu, 19 Jul 2007 17:02:59 +0200
Subject: USB: devices misc: Trivial patch to build the IOWARRIOR when it is selected in Kconfig
To: linux-usb-devel@lists.sourceforge.net
Message-ID: <200707191702.59834.jbe@pengutronix.de>
Content-Disposition: inline

From: Juergen Beisert <jbe@pengutronix.de>

Trivial patch to build the IOWARRIOR when it is selected in Kconfig.

Signed-off-by: Juergen Beisert <jbe@pengutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Makefile |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/Makefile
+++ b/drivers/usb/Makefile
@@ -48,6 +48,7 @@ obj-$(CONFIG_USB_SISUSBVGA)	+= misc/
 obj-$(CONFIG_USB_TEST)		+= misc/
 obj-$(CONFIG_USB_TRANCEVIBRATOR)+= misc/
 obj-$(CONFIG_USB_USS720)	+= misc/
+obj-$(CONFIG_USB_IOWARRIOR)	+= misc/
 
 obj-$(CONFIG_USB_ATM)		+= atm/
 obj-$(CONFIG_USB_SPEEDTOUCH)	+= atm/
